README.md

```diff
@@ -1,118 +1,19 @@
-# Code Style
-This project uses the Criteo Java code style. Please follow the instructions contained on this [page](https://go.crto.in/publisher-sdk-java-code-style)
+# Criteo Publisher SDK
+> Criteo Publisher SDK is meant for app publishers wanting to integrate in their app the Criteo Direct 
+> Bidder product. Direct Bidder for App allows you to monetize your mobile apps by accessing Criteo’s 
+> unique demand with a fast, lightweight and flexible integration. Choose between our App Bidding SDK 
+> or use our adapter for your mediation provider. Pick the method that applies to you and follow the 
+> [guide](https://publisherdocs.criteotilt.com/app/android/get-started/) to start the integration.
 
-# Testing
-## Tests organisation
+## Setup & Usage 
+Please refer to the [App Developer Portal](https://publisherdocs.criteotilt.com/app/android/),
+for detailed setup and usage instructions.
 
-The tests in this project are organised according to the following convention:
-- Unit tests are located within the [test](src/test/) directory.
-- Integration tests, which are written as instrumentation tests, are located within the [androidTest directory](src/androidTest)
-- The subset of integration tests which represent one of the functional tests defined [here](https://go.crto.in/publisher-sdk-functional-tests)
- are post-fixed with `FunctionTests`. The rest are post-fixed with `IntegrationTests`.
+## Contributing
+Pull requests are welcome. Please read [conventions and practices](DEVELOPMENT.md) beforehand.
```

Comment:
There is no DEVELOPMENT.md file

Reply:
Good catch. I think we should remove the contributing section for now. We will add it later once we figure it out.